static/nft-sample.svg

```diff
@@ -0,0 +1,27 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1000" height="1100" viewBox="0 0 1000 1110" style="background:#4a90e2">
+    <style>
+        text {
+            font: bold 30px sans-serif;
+            fill: rgba(255,255,255,0.8);
+        }
+    </style>
+
+    <text x="5" y="34">
+        Thousand Ether Homepage
+    </text>
+    <text x="1000" y="34" text-anchor="end">
+        #1234
+    </text>
+    <svg y="50" width="1000" height="1000" viewBox="0 0 100 100">
+        <rect width="100" height="100" fill="white" />
+        <image width="100" height="100" href="mainnet-grid.svg" opacity="0.1" />
```

Comment:
When testing locally this background image loads when I open the svg in my browser but doesn't when I embed it into an html page:

```html
<html>
  <body><img src="nft-sample.svg"></img></body>
</html>
```

This is using a local server not `file://` urls.